src/Lean/Server/InfoUtils.lean

```diff
@@ -86,6 +86,20 @@ where go ctx? a
     ts.foldl (init := a) (go <| i.updateContext? ctx?)
   | _ => a
 
+/-- Like `InfoTree.foldInfo`, but also passes the whole node to `f` instead of just the head. -/
```

Comment:
```suggestion
/--
Fold an info tree as follows, while ensuring that the correct `ContextInfo` is supplied at each stage:

 * Nodes are combined with the initial value `init` using `f`, and the result is then combined with the children using a left fold
 * Everything else returns the initial value

This is like `InfoTree.foldInfo`, but it also passes the whole node to `f` instead of just the head.
-/
```